django/api/scheduled_jobs.py

```diff
@@ -10,3 +10,9 @@ def schedule_get_ncda_redeemed_rebates():
     task_name = "api.tasks.check_rebates_redeemed_since"
     if not schedule_exists(task_name):
         schedule(task_name, None, task_name, schedule_type="D")
+
+
+def schedule_cancel_initiated_household():
+    task_name = "api.tasks.cancel_household_applications_initiated_status"
+    if not schedule_exists(task_name):
+        schedule(task_name, None, task_name, schedule_type="D")
```

Comment:
The 2nd and 3rd arguments (None, task_name) are args meant for your cancel_household_applications_initiated_status() function; since that function takes no arguments, you can delete these 2 args.